core/miner/main/main.cpp

```diff
@@ -70,6 +70,7 @@ namespace fc {
         codec::cbor::kDefaultT<Multiaddress>(), {}};
     boost::optional<Address> actor, owner, worker;
     boost::optional<RegisteredSealProof> seal_type;
+    std::vector<Address> precommit_control;
```

Comment:
Seems it is a config for mining and should be in `core/miner/storage_fsm/sealing.hpp`. I'm not sure about it, there is no comment what config this is. Please, ask @ortyomka for it.

Reply:
okay, I will contact him

Reply:
@ortyomka  said that this is okay for current situation